Atomic/mount.py

```diff
@@ -449,6 +460,12 @@ def _cleanup_container(self, cinfo):
             except Exception as e:
                 raise MountError(e)
 
+    def _clean_tmp_image(self):
+        # If a temporary image is created with commit,
+        # clean up that too
+        if self.tmp_image is not None:
+            self.client.remove_image(self.tmp_image)
+
```

Comment:
That is done in init by default -> https://github.com/projectatomic/atomic/pull/202/files#diff-ce5473cd96a720a53d0834698c9b2eaaR179